README.md

```diff
@@ -9,194 +9,192 @@
 
-Or by importing relevant FDC3 functions with ES6 module syntax:
+[FDC3 Charter]: https://fdc3.finos.org/docs/fdc3-charter
+[Compliance]: https://fdc3.finos.org/docs/fdc3-compliance
+[API Reference]: https://fdc3.finos.org/docs/api/ref/DesktopAgent
 
-```ts
-import { getOrCreateChannel } from '@finos/fdc3'
+# Supported Platforms
 
-// will throw if window.fdc3 is not available
-const channel = await getOrCreateChannel('myChannel')
-channel.broadcast({ ... })
-```
+ - As an open standard, FDC3 can be implemented on any platform and in any language.
+ - All that is required is a "desktop agent" that supports the FDC3 standard, which is responsible for co-ordinating application interactions.
+ - Get started using FDC3 on the web with typescript by reading the [supported platforms](https://fdc3.finos.org/docs/supported-platforms) page.
 
-## Native
-The FDC3 standard does not define wire formats for communication. Hence, for native applications to be FDC3-enabled, they need to make use of a library (e.g. a DLL in .Net or Jar file in Java) that provides them with an implementation of the FDC3 API. FDC3-enabled native applications are therefore specific to particular desktop container frameworks (or other suitable environments) that provide the necessary libraries.
+## FDC3 npm module
 
-Despite this limitation, implementing support for FDC3 in a native application can allow it to interact with a wide variety of FDC3-enabled web applications.
+The FDC3 npm package does NOT provide a Desktop Agent implementation. Rather, it can by used by web applications to target operations from the API Specification in a consistent way. Each FDC3-compliant desktop agent that the application runs in, can then provide an implementation of the FDC3 API operations.
 
-# Roadmap
+For installation and usage instructions, see: https://fdc3.finos.org/docs/supported-platforms#usage
 
-- [x] Publish versioned NPM packages to ease adoption and implementation of FDC3 APIs.
-- [x] Release version 1.2 of the standard with expanded API capabilities.
-- [x] Establish a process to accelerate community-contributed context data definitions.
-- [ ] Improve the app directory specification, with support for container-agnostic app manifests, and more type metadata around channels and launching apps.
-- [ ] Release version 2.0 of the standard with API support for app instances, two-way data flow and requesting channels with context.
-- [ ] Introduce a hosted app directory example with sample application definitions.
-- [ ] Continue building out business use cases to inform future versions of the FDC3 Standard.
+# Getting Involved
 
-# Using the standard? Let us know!
+## Using the standard? Let us know!
 
 If you are an existing individual or corporate user of the FDC3 standard, we would love to hear from you: just email [fdc3@finos.org](mailto:fdc3@finos.org) with details about how you are using the standard.
```

Comment:
@robmoffat should we mention this page at all https://www.finos.org/get-involved-fdc3 (aka you plan to keep it updated?) 

Or do you want to direct people to directly raise a PR against the community page? 

Your call.

Reply:
I think I'd rather get the email.   I've ask Niamh to take that page down - it's out-of-date and contained many of the wrong questions anyway.